src/Apps/Articles/ArticlesApp.tsx

```diff
@@ -16,7 +17,7 @@ const ArticlesApp: FC<ArticlesAppProps> = ({ viewer }) => {
   useScrollToOpenEditorialAuthModal()
 
   return (
-    <>
+    <AnalyticsContextProvider>
```

Comment:
Rather than dropping the analytics context provider everywhere, can we just put it into `Boot`?

Reply:
That's the trade off: you don't have access to the router within boot so you can't listen to route changes.

Reply:
Here's your spot (either AppShell, or that first route mount point -- could wrap that with a component that accepts children: https://github.com/artsy/force/blob/main/src/System/Router/buildAppRoutes.tsx#L42C11-L49

Reply:
Ah yeah I forgot about AppShell!

Reply:
Cool, updated; much better.

Reply:
👀